core/Network/TLS/Parameters.hs

```diff
@@ -36,6 +38,9 @@ import Network.TLS.X509
 import Network.TLS.RNG (Seed)
 import Data.Default.Class
 import qualified Data.ByteString as B
+#if __GLASGOW_HASKELL__ < 710
+import Data.Monoid (mempty)
+#endif
```

Comment:
Hey. 

I was rather trying to get the build green, instead of fixing warnings. But I may as well go all the way! 

I will use your suggestions and create a new pull request. Thanks. 😊